driver-core/src/test/java/com/datastax/driver/core/ProtocolV1Test.java

```diff
@@ -24,7 +25,7 @@
 /**
  * Tests targeting protocol v1 specifically.
  */
-@CCMConfig(version = "1.2.19")
+@CassandraVersion(major = 1.2, exactRequired = true)
```

Comment:
You need to hook in early enough to prevent the CCM cluster from being created. Try this:

```java
    @Override
    public void beforeTestClass(Object testInstance) throws Exception {
        if (CCMBridge.isWindows())
            throw new SkipException("Only works on proper OS's");
        super.beforeTestClass(testInstance);
    }
```

(assuming `CCMBridge.isWindows()` is made accessible)